README.md

```diff
@@ -0,0 +1,65 @@
+# Reverse chain adapter
+## Service overview 
+Reverse chain adapter is a service that is used for handling crucial financial operations in Iroha Hyperledger. 
+It uses RabbitMQ as middleware between Iroha clients and Iroha peer. 
+Using the service, transactions are not sent directly to Iroha peer but stored in a dedicated persistent queue. 
+Once a transaction is saved in the queue, the service will read the incoming transaction and try to send it to Iroha peer. 
```

Comment:
Once a transaction is saved in the queue, the service will read that transaction and try to send it to Iroha peer.